tabs/internal/tabs.ts

```diff
@@ -63,6 +63,7 @@ export class Tabs extends LitElement {
    *
    * @export
    */
+  @property({type: Number, reflect: true})
```

Comment:
This property can't reflect since its value is dynamically updated, similar to [select's `selectedIndex` property](https://github.com/material-components/material-web/blob/6b95a13ee1fd4203140aeda89c24c7afb8acdde1/select/internal/select.ts#L194-L199). It also needs to use kebab-case for its attribute. Can you update it to this?

```ts
@property({type: Number, attribute: 'active-tab-index'})
```

Reply:
Sure, I'm pretty new to lit, so sorry for the error. I just did what worked for my limited example.
Changed ✅

Reply:
No worries, thanks for the update! I'll pull this in and let ya know if we need any other changes.